eng/Packages.Data.props

```diff
@@ -9,11 +9,18 @@
   <ItemGroup>
     <PackageReference Update="ApprovalTests" Version="3.0.22" />
     <PackageReference Update="ApprovalUtilities" Version="3.0.22" />
+    <PackageReference Update="Azure.Data.AppConfiguration" Version="1.0.0" />
     <PackageReference Update="Azure.Core" Version="1.0.1" />
     <PackageReference Update="Azure.Identity" Version="1.1.0" />
     <PackageReference Update="Azure.Security.KeyVault.Secrets" Version="4.0.0" />
+    <PackageReference Update="Azure.Security.KeyVault.Certificates" Version="4.0.0" />
+    <PackageReference Update="Azure.Security.KeyVault.Keys" Version="4.0.1" />
     <PackageReference Update="Azure.Messaging.EventHubs" Version="5.0.0-preview.6" />
-    <PackageReference Update="Azure.Storage.Blobs" Version="12.0.0" />
+    <PackageReference Update="Azure.Storage.Blobs" Version="12.2.0" />
```

Comment:
Updated all to use latest version as otherwise I got a downgrade detected error.
@jsquire  any impact on updating the version of Blobs for Event Hubs?

Reply:
Our use is pretty basic, so I suspect it's safe.  If you do a CI run with Event Hubs, the live tests would surface any issues.

Reply:
Yes - nothing changed that should affect Event Hubs and it'd be great to target the latest Storage when you GA.